### PR TITLE
Add option for a `.levignore.js` file to ignore specific changes in the report

### LINKED ISSUE #306: feat: add ability to ignore particular changes
The beginning of`.levignore`.

Currently we use levitate in Grafana and it creates a number of false positives as a result feature toggle related changes. When a feature is demonstrated stable and we no longer need the feature flag (so the feature will become GA), we need to remove it. This is not a breaking change, it's in practice an additional change, but levitate detects it as a breaking change because the feature flag is removed from the API.

We want to be able to pass parameters to ignore changes like this and then change our workflow in the grafana repository as appropriate.

### PR DESCRIPTION
**Pull request by academo**
**What this PR does / why we need it**:

Adds the ability to create a `.levignore.js` file in the execution directory to ignore specific changes, additions or removals in specific exports. 

The format of this file is as follows:

```javascript
 {
   "removals": [
     // each entry is a regex
     /Sample\.ignoreThisOne/,
     /Sample\.ignoreThisOneToo/,
     /Sample.*\b/,
     "Sample.ignoreThisOneToo" // strings are converted to regex to match exact
   ],
   "changes": [
     //...
   ],
   "additions": [
     //...
   ]
  }
```

**Which issue(s) this PR fixes**:

Closes https://github.com/grafana/levitate/issues/306

**Special notes for your reviewer**:

### CODE REVIEW

File: scripts/list-exports-comparison.ts [hunk 1]
```diff
@@ -46,9 +46,9 @@ async function main() {
   }
 }
 
-function extractList(stdout: string) {
+function extractList(stdout: string): string[] {
   const lines = stdout.split('\n');
-  const final = [];
+  const final: string[] = [];
```
Comment (academo) [inline scripts/list-exports-comparison.ts:51]: This is a type fix from the typescript update.

File: src/integration-tests/levitate.test.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import path from 'path';
+import execa from 'execa';
+
+const nodeBinary = process.execPath;
+const levitateBinary = path.resolve(__dirname, '../../dist/bin.js');
+
+describe('Levitate', () => {
```
Comment (academo) [inline src/integration-tests/levitate.test.ts:7]: I added these basic integration tests mostly to make sure the levignore file works as expected but also to start creating more integration test for other commands (hence the help test basic tests)

File: src/utils.ts [hunk 3]
```diff
@@ -7,3 +9,64 @@ export function pathExists(path: string): Promise<boolean> {
...
+ *  "changes": [
+ *    //...
+ *  ],
+ *  "additions": [
+ *    //...
+ *  ]
+ * }
+ *
+ *
+ */
```
Comment (jackw) [inline src/utils.ts:33]: I would suggest moving this to the README to make it easier to find.

**Review by jackw — APPROVED** [review]
Code LGTM. 🚀 

Left a suggestion to add some docs to the Readme so before it makes its way to production there is at least something for others to reference without needing to look in the code.

File: src/utils.ts (hunk 3 above)
Comment (academo) [reply src/utils.ts:33]: Good idea. I added a section in the readme about it.

**Pull request merged**